R/PipeOpLearnerQuantiles.R

```diff
@@ -0,0 +1,228 @@
+#'   Identifier of the resulting  object, internally defaulting to the `id` of the [`Learner`][mlr3::Learner] being wrapped.
+#' * `param_vals` :: named `list`\cr
+#'   List of hyperparameter settings, overwriting the hyperparameter settings that would otherwise be set during construction. Default `list()`.
+#'
+#' @section Input and Output Channels:
+#' [`PipeOpLearnerQuantiles`] has one input channel named `"input"`, taking a [`TaskRegr`][mlr3::TaskRegr] specific to the [`Learner`][mlr3::Learner]
+#' type given to `learner` during construction; both during training and prediction.
+#'
+#' [`PipeOpLearnerQuantiles`] has one output channel named `"output"`, producing `NULL` during training and a [`PredictionRegr`][mlr3::Prediction] object
+#' during prediction.
+#'
+#' The output during prediction is a [`PredictionRegr`][mlr3::PredictionRegr] on the prediction input data that aggregates all `result`s produced by the [`Learner`][mlr3::Learner] for each quantile in `quantiles`.
+#' trained on the training input data.
+#'
+#' @section State:
+#' The `$state` is set during training. It is a named `list` with the member:
+#' * `model_states` :: `list`\cr
+#'   List of the states of all models created by the [`Learner`][mlr3::Learner]'s `$.train()` function.
+#'
+#' @section Parameters:
+#' The parameters are exactly the parameters of the [`Learner`][mlr3::Learner] wrapped by this object.
+#' * `q_vals` :: `numeric`\cr
+#'   Quantiles to use for training and prediction.
+#'   Initialized to `c(0.05, 0.5, 0.95)`
+#'
+#' * `q_response` :: `numeric(1)`\cr
+#'   Which quantile in `quantiles` to use as a `response` for the [`PredictionRegr`][mlr3::PredictionRegr] during prediction.
+#'   Initialized to `0.5`.
+#'
+#' @section Internals:
+#' The `$state` is updated during training.
+#'
+#' @section Fields:
+#' Fields inherited from [`PipeOp`], as well as:
+#' * `learner` :: [`LearnerRegr`][mlr3::LearnerRegr]\cr
+#'   [`Learner`][mlr3::Learner] that is being wrapped. Read-only.
+#' * `learner_model` :: [`Learner`][mlr3::Learner]\cr
+#'   If [`PipeOpLearnerQuantiles`] has been trained, this is a `list` containing the [`Learner`][mlr3::Learner]s for each quantile.
+#'   Otherwise, this contains the [`Learner`][mlr3::Learner] that is being wrapped.
+#'   Read-only.
+#' * `predict_type`\cr
+#'   Predict type of the [`PipeOpLearnerQuantiles`], which is always `"response"  "quantiles"`.
+#'
+#' @section Methods:
+#' Methods inherited from [`PipeOp`].
+#'
+#' @family PipeOps
+#' @family Meta PipeOps
+#' @template seealso_pipeopslist
+#' @include PipeOp.R
+#' @export
+#' @examples
+#' library("mlr3")
+#'
+#' task = tsk("boston_housing")
+#' learner = lrn("regr.debug")
+#' po = mlr_pipeops$get("learner_quantiles", learner)
+#'
+#' po$train(list(task))
+#' po$predict(list(task))
+PipeOpLearnerQuantiles = R6Class("PipeOpLearnerQuantiles",
+  inherit = PipeOp,
+  public = list(
+   initialize = function(learner, id = NULL, param_vals = list()) {
```

Comment:
sorry to be super pedantic, but could you check the indentation here? It should always be +2 spaces for each level of indentation

Reply:
No problem, I changed indentation to 2 spaces in the whole file.